crates/sui-deepbook-indexer/src/server.rs

```diff
@@ -61,3 +82,59 @@ async fn get_pools(
 
     Ok(Json(results))
 }
+
+async fn get_24hr_volume(
+    Path(pool_id): Path<String>,
+    State(state): State<PgDeepbookPersistent>,
+) -> Result<Json<i64>, DeepBookError> {
+    let connection = &mut state.pool.get().await?;
+    let unix_ts = SystemTime::now()
+        .duration_since(UNIX_EPOCH)
+        .unwrap()
+        .as_millis() as i64;
+    let day_ago = unix_ts - 24 * 60 * 60 * 1000;
+    let results: Vec<OrderFill> = schema::order_fills::table
+        .select(OrderFill::as_select())
+        .filter(schema::order_fills::pool_id.eq(pool_id))
+        .filter(schema::order_fills::onchain_timestamp.gt(day_ago))
+        .load(connection)
+        .await?;
+
+    let mut total_vol = 0;
+    for order_fill in results {
+        total_vol += order_fill.base_quantity;
+    }
```

Comment:
why not using `sum`? 

this is from claude:
```
let total_vol: i64 = schema::order_fills::table
    .select(sum(schema::order_fills::base_quantity))
    .filter(schema::order_fills::pool_id.eq(pool_id))
    .filter(schema::order_fills::onchain_timestamp.gt(day_ago))
    .first(connection)
    .await?
    .unwrap_or(0);  // In case there are no results, default to 0
```